main.go

```diff
@@ -28,7 +28,7 @@ var (
 	socket *pipe.Pipe
 
 	mask = gocv.NewMat()
-	rect = image.Rect(640, 0, 1400, 500)

```

Comment:
Can you leave this line the same and inside loadSwitch can you rewrite it like this?